src/app/modules/sgid/sgid.constants.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Name of cookie which contains state of sgid login, and access
+ * token if login was successful.
+ */
+export const SGID_COOKIE_NAME = 'jwtSgid'
```

Comment:
I noticed this string is hardcoded elsewhere in the backend; if we are making this change in this PR, can we use this constant there too?

Reply:
thanks, made the changes